src/app/loadout-builder/filter/PickerSectionMods.tsx

```diff
@@ -7,74 +7,64 @@ import { SelectableArmor2Mod } from '../locked-armor/SelectableBungieImage';
 import { LockedArmor2Mod, ModPickerCategories, ModPickerCategory } from '../types';
 import styles from './PickerSection.m.scss';
 
+/** Slot specific mods can have at most 2 mods. */
+const MAX_SLOT_SPECIFIC_MODS = 2;
+/** Raid, combat and legacy mods can have up to 5 selected. */
+const MAX_SLOT_INDEPENDENT_MODS = 5;
+
 export default function PickerSectionMods({
   defs,
   mods,
   locked,
   title,
   category,
-  maximumSelectable,
-  energyMustMatch,
-  splitByItemTypeDisplayName,
   onModSelected,
   onModRemoved,
 }: {
   defs: D2ManifestDefinitions;
   mods: readonly LockedArmor2Mod[];
-  locked?: readonly LockedArmor2Mod[];
+  locked: readonly LockedArmor2Mod[];
   title: string;
   category: ModPickerCategory;
-  maximumSelectable: number;
-  energyMustMatch?: boolean;
-  splitByItemTypeDisplayName: boolean;
   onModSelected(mod: LockedArmor2Mod);
   onModRemoved(mod: LockedArmor2Mod);
 }) {
   if (!mods.length) {
     return null;
   }
   const lockedModCost = _.sumBy(locked, (l) => l.modDef.plug.energyCost?.energyCost || 0);
-  const isNotGeneralOrOtherOrRaid =
+  const isSlotSpecificCategory =
```

Comment:
Would it be easier to enumerate the slot specific ones, rather than listing out all the non-slot-specific mod categories?

Reply:
Yeah it would be safer for future changes too. Will do.